CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/CMakeModules/Modules)
 ###############################################################################
 
 # Valid applications and choices
-list(APPEND VALID_APPS ATM ATMW S2S S2SW DATM DATM_NEMS)
+list(APPEND VALID_APPS ATM ATMW S2S S2SW NG-GODAS NG-GODAS-NEMSDATM)
```

Comment:
No changes have been made in the NEMS driver to switch these names---it will still use "FRONT_NEMS_DATM" and "FRONT_CDEPS_DATM" which then asks for "model" names ``nems_datm`` or ``datm``. It seems we're introducing a lot of inconsistency.

Reply:
@DeniseWorthen You are correct.

Reply:
@DeniseWorthen My understanding is that this is actually to avoid confusion. in NEMS driver we specify the subcomponent models, not the applications names. Using DATM or NEMS_DATM for both subcomponent and application names could be confusing. Here the NG_GODAS is application name, this application will still define the subcomponents DATM (cdeps datm), MOM6 and CICE6 etc. While before we use DATM as application name and use DATM also as DATM subcomponent along with other subcomponents.

Reply:
Are both `NG-GODAS` and `NG-GODAS-NEMSDATM` needed? What's the difference?

Reply:
NG-GODAS refers to "mom6_cice6_cdeps_datm" application.
NG-GODAS-NEMSDATM refers to "mom_6_cice6_nems_datm" application.

Reply:
OK. But do we need both? Didn't we switch to CDEPS, why do we need nems? Are the results different when CDEPS is used compared to NEMS? Are different atm data used?

Reply:
In the future, NEMS_DATM will be removed from ufs-weather-model. See a related issue at https://github.com/ufs-community/ufs-weather-model/issues/565.

Reply:
@junwang-noaa I see, thanks. I do see how DATM is being used two ways (app and component) so this is probably better.